setup/updating.md

```diff
@@ -37,12 +37,20 @@ Updating Coder is a two-step process:
     ```bash
     helm repo update
     ```
+    
+2. Export the current helm chart values into a file:
 
-2. Upgrade to the desired version (i.e., `1.16.0`):
+    ```bash
+    helm get values --namespace coder coder > current-values.yml
```

Comment:
suggest making this an optional step, the preferred flow (IMO) should be that an admin already has their values file (because they version controlled it), so this step is only needed if they _gasp_ forgot to do that when they installed/etc

Reply:
good point!